src/vsr/replica.zig

```diff
@@ -4749,7 +4648,7 @@ pub fn ReplicaType(
                     assert(message.header.replica == self.replica);
                     assert(message.header.op == self.op);
                     assert(message.header.commit == self.commit_min);
-                    assert(replica == self.primary_index(self.view));
```

Comment:
Why do we delete this assert? It should still hold, no?

Reply:
Ah that snuck in via `git add --patch`. I'm ok leaving it removed b/c it will very shortly not be true, since DVCs will be broadcast to all replicas.